src/Symfony/Component/Workflow/Dumper/DumperInterface.php

```diff
@@ -25,10 +25,6 @@ interface DumperInterface
     /**
      * Dumps a workflow definition.
      *
-     * @param Definition   $definition A Definition instance
-     * @param Marking|null $marking    A Marking instance
-     * @param array        $options    An array of options
-     *
```

Comment:
This should be done on a lower branch.

Reply:
Let's minimize the changes that don't add value to the code. Keeping that in lower branches is fine by me.